zerver/lib/markdown/__init__.py

```diff
@@ -2175,15 +2175,17 @@ def build_inlinepatterns(self) -> markdown.util.Registry:
         # emphasis2 -       we disable _ for bold and emphasis
 
         # Declare regexes for clean single line calls to .register().
-        NOT_STRONG_RE = markdown.inlinepatterns.NOT_STRONG_RE
+        #
         # Custom strikethrough syntax: ~~foo~~
         DEL_RE = r"(?<!~)(\~\~)([^~\n]+?)(\~\~)(?!~)"
         # Custom bold syntax: **foo** but not __foo__
         # str inside ** must start and end with a word character
         # it need for things like "const char *x = (char *)y"
-        EMPHASIS_RE = r"(\*)(?!\s+)([^\*^\n]+)(?<!\s)\*"
-        ENTITY_RE = markdown.inlinepatterns.ENTITY_RE
-        STRONG_EM_RE = r"(\*\*\*)(?!\s+)([^\*^\n]+)(?<!\s)\*\*\*"
+        EMPHASIS_RE = r"(\*)(?!\s+)([^\*^\n]+?)(?<!\s)\2"
```

Comment:
This is not the same value we had before.  It's fine to make changes to how it's encoded, but please do so in a separate commit from one that is described as just moving code around without changes.

(Same with `STRONG_EM_RE`).